engine/apps/schedules/models/shift_swap_request.py

```diff
@@ -164,7 +164,15 @@ def status(self) -> str:
         return self.Statuses.OPEN
 
     @property
-    def slack_channel_id(self) -> str | None:
+    def slack_channel(self) -> typing.Optional["SlackChannel"]:
+        """
+        This is only set if the schedule associated with the shift swap request
+        has a Slack channel configured for it.
+        """
+        return self.schedule.slack_channel
```

Comment:
added for [this](https://github.com/grafana/oncall/pull/5292/files#diff-505c54140ab7b80107fe3845db9554bf9434a10f57fae4335f60eab72d366d91R169)